pyart/retrieve/kdp_proc.py

```diff
@@ -21,12 +33,978 @@
 
 import time
 import numpy as np
+from functools import partial
+
 
-from scipy import optimize, stats
+SCALERS = [0.1,10**(-0.8),10**(-0.6), 10**(-0.4), 10**(-0.2), 1,
+           10**(-0.2), 10**(-0.4), 10**(-0.6), 10**(-0.8),1, 10]
+           
+from scipy import optimize, stats, interpolate, linalg, signal
 
 from . import _kdp_proc
 from ..config import get_field_name, get_metadata, get_fillvalue
 
+def kdp_schneebeli(radar, gatefilter=None, fill_value = None, psidp_field=None,
+                kdp_field=None, phidp_field=None, band = 'C', rcov = 0, pcov = 0,
+                prefilter_psidp = False, filter_opt = {}, parallel = True):
```

Comment:
The default value of filter_opt should be `None`.  Using a mutable default type [is dangerous](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)